apps/src/levelbuilder/unit-editor/UnitEditor.jsx

```diff
@@ -78,10 +79,15 @@ class UnitEditor extends React.Component {
     initialLocales: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string))
       .isRequired,
     initialProjectSharing: PropTypes.bool,
-    initialCurriculumUmbrella: PropTypes.oneOf(
-      Object.values(CurriculumUmbrella).push('')
-    ),
+    initialCurriculumUmbrella: PropTypes.oneOf([
+      ...Object.values(CurriculumUmbrella),
+      '',
+    ]),
     initialTopicTags: PropTypes.arrayOf(PropTypes.string),
+    initialContentArea: PropTypes.oneOf([
+      ...Object.keys(CurriculumContentArea),
```

Comment:
This is deviating from the pattern for curriculum umbrella by design as Natalia suggested its easier for queries when the data is in snake case, hence using the keys from the dictionary.

Reply:
Maybe put a comment about this in the code so that we know the reason for the difference and will pick the right thing to do if we add another field here?

Reply:
Ya, that's a good idea.